firebase-firestore/src/jsMain/kotlin/dev/gitlive/firebase/firestore/firestore.kt

```diff
@@ -57,6 +52,28 @@ actual class FirebaseFirestore(val js: firebase.firestore.Firestore) {
         rethrow { js.clearPersistence().await() }
 
     actual fun useEmulator(host: String, port: Int) = rethrow { js.useEmulator(host, port) }
+
+    actual fun setSettings(value: FirebaseFirestoreSettings) {
+        value.run {
+            persistenceEnabled?.let { js.enablePersistence() }
+
+            val settingsJson = mutableListOf<Pair<String, Any>>()
```

Comment:
dont need the list just do `= json()` here
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.js/-json/